### PR TITLE
Upgrade dependencies to fix audit issues

### PR DESCRIPTION
**Pull request by mrtraser**
- change imports for ***.md** files in stories
- remove unneeded dependencies

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-294.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-294.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-294.gearboxjs.preview.cogniteapp.com

File: src/components/TimeseriesSearch/stories/TimeseriesSearch.stories.tsx [hunk 1]
```diff
@@ -80,6 +80,9 @@ storiesOf('TimeseriesSearch', module).add(
   'Full Description',
   () => {
     setupMocks();
+
+    console.log(fullDescription);
```
Comment (xitman78) [inline src/components/TimeseriesSearch/stories/TimeseriesSearch.stories.tsx]: remove it

Comment (mrtraser) [reply src/components/TimeseriesSearch/stories/TimeseriesSearch.stories.tsx]: done

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-294.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Closes #295

**Pull request merged**